rest-service/manager_rest/storage/resource_models.py

```diff
@@ -395,6 +398,11 @@ def response_fields(cls):
         fields['deployment_groups'] = flask_fields.List(flask_fields.String)
         return fields
 
+    @classproperty
+    def allowed_filter_attrs(cls):
+        return ['blueprint_id', 'created_by', 'description', 'site_name',
+                'latest_execution_status']
```

Comment:
`latest_execution_status` doesn't exist

Reply:
I thought it would be worth adding it now. But I'll wait for @mabuaisha changes.